selene_sdk/targets/genomic_features.py

```diff
@@ -234,6 +235,12 @@ class GenomicFeatures(Target):
                                  input the feature name and returns\
                                  the feature's threshold.
 
+    init_unpicklable : bool, optional
+        Default is False. If False, delay part of initialization code
```

Comment:
this might be a dumb question, but when should `init_unpicklable` be true? based on this docstring, if you don't need parallel processing, do you set this to `True` and then it's.... faster to use?

Reply:
this is just added in case it is needed for convenience? this is probably only needed if people are writing some custom code not using the CLI, like if people want to access the `data` attribute after initialization, it might be annoying if they have to call some method to make it initialized first. In typical usage, setting init_unpicklable to true or false should not make a big difference.